packages/kbn-shared-ux-components/src/empty_state/no_data_views/no_data_views.tsx

```diff
@@ -11,12 +11,10 @@ import React, { useCallback, useEffect, useRef } from 'react';
 import { useEditors, usePermissions, useDocLinks } from '@kbn/shared-ux-services';
 import type { SharedUxEditorsService } from '@kbn/shared-ux-services';
 
-import { DataView } from '../../../../../data_views/public';
-
 import { NoDataViews as NoDataViewsComponent } from './no_data_views.component';
 
 export interface Props {
-  onDataViewCreated: (dataView: DataView) => void;
+  onDataViewCreated: (dataView: unknown) => void;
```

Comment:
😬

Reply:
Yep, I know.  I'm filing a follow-up issue for this, so we can resolve it.  I'll add the comment I made in https://github.com/elastic/kibana/pull/127546